### DISCUSSION TITLE
3D Tiles with Meshes?

### DISCUSSION THREAD

**Discussion by DerKarlos** [StarArawn/bevy_ecs_tilemap#186] [Q&A]
I like the concept of hexagonal tiles and would like to use it.
Would the plugin also work with 3D rendering?
Could I place variations of complex 3D Meshes on the "tiles"?
An example would be great.

**Comment by StarArawn** [comment]
Hi!

This plugin is strictly for 2D tile maps. For a 3D tile map that uses a collection of meshes to display tiles I would use something custom. Likely you would want to use instanced rendering and the rendering for that kind of tile map would be vastly different. Some of the techniques that I use in this library could transfer over. If you're considering creating something like this feel free to use code from this library! 👍
